docs/release-1.2/existing-rhdh.md

```diff
@@ -64,121 +91,156 @@ metadata:
 EOF
 ```
 The value of `.data.npmrc` points to https://npm.registry.redhat.com.
-For testing RC plugin versions, update to `cmVnaXN0cnk9aHR0cHM6Ly9ucG0uc3RhZ2UucmVnaXN0cnkucmVkaGF0LmNvbQo=` (points to https://npm.stage.registry.redhat.com and can be accessed internally). If there is a need to point to multiple registries, modify the content of the secret's data from:
-```
+For testing RC plugin versions, update to `cmVnaXN0cnk9aHR0cHM6Ly9ucG0uc3RhZ2UucmVnaXN0cnkucmVkaGF0LmNvbQo=` (which points to https://npm.stage.registry.redhat.com and can be accessed internally within Red Hat). If there is a need to point to multiple registries, modify the content of the secret's data from:
+```yaml
   stringData:
     .npmrc: |
       registry=https://npm.registry.redhat.com
 ```
 to
-```
+```yaml
   stringData:
     .npmrc: |
       @redhat:registry=https://npm.registry.redhat.com
       @<other-scope>:registry=<other-registry>
 ```
 
 ### dynamic-plugins ConfigMap
-This ConfigMap houses the configuration for enabling and configuring dynamic plugins. To incorporate the orchestrator plugins, append the following configuration to the **dynamic-plugins** ConfigMap:
+This ConfigMap houses the configuration for enabling and configuring dynamic plugins in RHDH. 
+
+> Note: Be sure to review [this section](#identify-latest-supported-plugin-versions) to determine the latest supported Orchestrator plugin `package:` and `integrity:` values, and update the dynamic-plugin ConfigMap entries accordingly. The samples in this document may not be the latest. 
```

Comment:
Would it make sense to provide a link so users know where to check?

Reply:
`[this section](#identify-latest-supported-plugin-versions)` links to the section in this doc - or are you referring to something else?